freeipa/templates/deployment.yaml

```diff
@@ -66,6 +66,10 @@ spec:
           - --ssh-trust-dns
           {{- end }}
           ports:
+          - containerPort: 389
+            protocol: TCP
```

Comment:
Please do not expose the port 389. Always use the secure port.

Reply:
done